.pre-commit-config.yaml

```diff
@@ -37,7 +37,7 @@ repos:
       # You are encouraged to use static refs such as tags, instead of branch name
       #
       # Running "pre-commit autoupdate" would automatically updates rev to latest tag
-      rev: 0.13.1+ibm.56.dss
+      rev: 0.13.1+ibm.57.dss
```

Comment:
Should this be `0.13.1+ibm.58.dss` also? ... I see you put `0.13.1+ibm.58.dss` in `detect_secrets/__init__.py` below.

Reply:
No, because the `0.13.1+ibm.58.dss` version hasn't been officially released yet (it won't be until this PR is merged into master). I'll create a separate PR to update the baseline to use that version after it's released.

Reply:
> I see you put 0.13.1+ibm.58.dss in detect_secrets/__init__.py below.

I bumped the version in that file so that the master build will know to release a new version